### ISSUE TITLE
📚 Documentation: Heading levels in menu should be h2s [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#279]
## Issue
All headings in the left navigation should be changed to `<h2>`s. 

Ideally these would come after an `<h1>` but due to current page design it is acceptable compromise to have them come before as a quick fix for now.

![side nav with a heading highlighted, console showing elements pane indicating it is a <h4>](https://github.com/appwrite/website/assets/121672086/aea8f451-a522-4d1e-bbd4-79e8b96eb45d)

## Why it matters
People using assistive technology rely on proper headings structure to navigate effectively. By skipping levels people can get lost or miss entire sections of content using assistive tech.

## Suggested fix
All headings in the left navigation should be changed to `<h2>`s. 

## Relevant Success Criterion WCAG
[Info and Relationships in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
[Technique using heading levels](https://www.w3.org/WAI/WCAG21/Techniques/html/H42)

## Additional Notes
Ideal First issue.

[Found on stream on 2023-10-25](https://www.twitch.tv/videos/1959992199)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Arindam200** [comment]
I would love to work on this! Can you please assign me?

**Comment by CBID2** [comment]
I would love to work on this issue :)

**Comment by uttampun44** [comment]
I would like to fix the issue. Assign me

**Comment by TGlide** [comment]
@Arindam200 assigned it to you 😄

**Comment by Arindam200** [comment]
> @Arindam200 assigned it to you 😄

Thanks! I have made the PR
